packages/guides-graphs/src/Graphs/DependencyInjection/GraphsExtension.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Graphs\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+use function dirname;
+
+class GraphsExtension extends Extension implements PrependExtensionInterface, CompilerPassInterface
```

Comment:
We can just skip the implementation of `PrependExtensionInterface` and `CompilerPassInterface` if those methods remain empty.